analyze.js

```diff
@@ -33,15 +33,13 @@ function analyze(options, callback){
 
   command = buildCommand(options);
 
-  var outPutFileName = (options.url) ? options.url.replace(/^(https?|ftp):\/\//, '') + '.json' : 'results.json';
-
-  exec(command + ' --reporter=json > ' + outPutFileName, function(error, stdout, stderr){
+  exec(command + ' --reporter=json > ' + options.outputFileName, function(error, stdout, stderr){
```

Comment:
@janjaap is the same https://github.com/rejas/gulp-louis/blob/602a71fe3a697e347d74ab2d6dfa48529fb8b7a2/louis.js#L27-L28 ?